src/components/Footer/styles.css

```diff
@@ -0,0 +1,521 @@
+@import url('https://rsms.me/inter/inter.css');
+html { font-family: 'Inter', sans-serif; }
+@supports (font-variation-settings: normal) {
+  html { font-family: 'Inter var', sans-serif; 
+    background: #F2F2F2;}
+}
+
+footer {
+    background: #FFFFFF;
+}
+
+#footer-component {
+    width: 100%;
+    display: grid;
+    justify-content: center;
+    text-align: center;
+    background: #FFFFFF;
+    margin: 0px auto 0px auto;
+    bottom:0; 
+}
+
+#become-nanny {
+    width: 473.8px;
+    height: 24px;
+    margin: 0 0 20px;
+    font-family: Inter;
+    font-size: 28px;
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 0.86;
+    letter-spacing: normal;
+    text-align: center;
+    color: #4a4a4a;
+}
+
+#become-nanny-h1 {
+    margin-top: 4rem;
+}
+
+#become-nanny-mobile {
+    display: none;
+}
+
+#become-nanny-h1-mobile {
+    display: none;
+}
+
+#footer-component p {
+    width: 362px;
+    height: 21px;
+    margin: 0px 65px 39px 65px;
+    font-family: Inter;
+    font-size: 19px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    text-align: center;
+    color: #4a4a4a;
+}
+
+ #footer-component button {
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+    align-items:  center;
+    color: #FFFFFF;
+    height: 70px;
+    width: 336px;
+    background: #7223b9;
+    border-radius: 34px;
+    border: none;
+    padding-left: 1.2rem;
+    margin: 0 auto 2rem auto;
+
+}
+
+#footer-button img {
+    width: 33px;
+    height: 30px;
+    margin: 5px 13px 1px -240px;
+    padding: 11.3px 2.5px 2.5px;
+    object-fit: contain;
+ }
+
+ #footer-button #create-nanny {
+    width: 219.6px;
+    height: 24.5px;
+    margin: -42px 0 8px 70px;
+    font-family: Inter;
+    font-size: 18px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.33;
+    letter-spacing: normal;
+    text-align: left;
+    color: #ffffff
+}
+
+#footer-button #take-less {
+    width: 146.1px;
+    height: 24.3px;
+    margin: -17px 65px 0 70px;
+    font-family: Inter;
+    font-size: 11.5px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1;
+    letter-spacing: normal;
+    text-align: left;
+    color: #ffffff;
+}
+
+footer a {
+    width: 200px;
+    height: 20px;
+    margin: -5px 138.6px 0 139px;
+    padding-left: 20px;
+    font-family: Inter;
+    font-size: 14px;
+    font-weight: normal;
```

Comment:
the `a` tag already has a font-weight: normal so you don't need to declare it here